lib/contestant.rb

```diff
@@ -0,0 +1,24 @@
+class Contestant
+  attr_reader :first_name, :last_name, :age,
+              :state_of_residence, :spending_money, :game_interest
+    def initialize (info)
+      @first_name = info[:first_name]
+      @last_name = info[:last_name]
+      @age = info[:age]
+      @state_of_residence = info[:state_of_residence]
+      @spending_money = info[:spending_money]
+      @game_interest = []
```

Comment:
i'd probably call this `@game_interests` -- since it's a collection, you're expecting it to potentially hold more than one interest, so plural is better for readability.